docs/src/components/DocLayout/TopBar.tsx

```diff
@@ -49,10 +48,16 @@ const TopBar = ({
   tabs,
   location,
   tocHasItems,
+  hasHeaderLink,
   headerLink,
   breadcrumbs,
   description,
+  hasStorybook,
+  storybookLink,
 }) => {
+  const hasTabs = tabs && tabs.length > 1;
```

Comment:
If the page only has one tab we don't need to display that as a tab (it was even displaying just an empty blank tab, not the actual title of the tab), because if there is no navigation to other tabs, why display tabs at all?

Reply:
there is might be a case, when it released only in Figma and have guidelines tab, but not react

Reply:
Right, missed that case. Added